### DISCUSSION TITLE
Why one trace has 2 records

### DISCUSSION THREAD

**Discussion by snowwolf007cn** [apache/skywalking#8074] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/985278/140272791-f9988feb-c441-4a04-8f46-b8b3fc3df784.png)
![image](https://user-images.githubusercontent.com/985278/140272851-9c128cdb-d76f-4137-97f7-d78328e849da.png)
both of the image above is the trace of api "/api/batch_add_customer/batch/distribute", they have the same trace id, one displayed error status and the other not? Is it some kind of bug?

**Comment by 1393467261** [comment, marked as answer]
There are multiple segments in one single trace，what you see is segment.
